setup.py

```diff
@@ -34,7 +34,7 @@ def find_version(*file_paths):
     'transformers>=4.3.0',
     'pandas',
     'pylint',
-    'sklearn',
+    'scikit-learn',
     'ogb',
```

Comment:
Should have a `>=` for the ogb version here as well since we had problems before?

Reply:
Can we fix it with ogb == 1.3.6?
Furthermore, we would better add torch >= 1.12

Reply:
Yeah if it's going to be more than one change let's open another issue and PR to keep this one focused.

Reply:
Opened #90 , if you can think of more _minimum_ requirements let's add them. I would suggest against pinning specific versions unless it's absolutely necessary as that would make installation for end users more difficult. "Larger or equal" tends to be good enough.

We can pin dependencies in our Dockerfiles to ensure working environment.